Templates/Deploy_Pipeline/Jenkins/Jenkinsfile

```diff
@@ -0,0 +1,222 @@
+def JenkinsAgent              = "ztec-201-STC"
+def pipeverbose               = true
+
+pipeline {
+    agent { label JenkinsAgent }
+
+    environment {
+        GIT_STRATEGY = "none"
+        GIT_CHECKOUT = "false"
+        wdEnvironmentFile = "EOLEB7-${params.application}-${params.targetEnvironment.capitalize()}.yaml" //Make sure the referenced file matches the required naming convention
+        wdEvidencePath = "deploy/evidences"
+        wdEvidencesRoot = "/var/work/wazi_deploy_evidences_jenkins/"
+        wdEvidencesIndex = "/var/work/wazi_deploy_evidences_jenkins_index/"
+        uniqueWorkspaceId = "${env.WORKSPACE}/${params.application}/deploy-${params.buildId}"
+        deployOutputDirectory = "${env.WORKSPACE}/${params.application}/DEPLOY-OUTPUT"
+    }
+
+    parameters {
+        string(name: 'application', defaultValue: '', description: 'Application name (e.g., retirementCalculator, MortgageApplication)')
+        string(name: 'releaseVersion', defaultValue: '', description: 'Release version to deploy (only for release eg: rel-1.6.1 )')
+        string(name: 'buildId', defaultValue: '', description: 'Build pipeline ID (e.g., 00000018)')
+        choice(name: 'targetEnvironment', choices: ['integration', 'acceptance', 'production'], description: 'Deployment target environment')
+        string(name: 'branchName', defaultValue: 'main', description: 'Branch name (e.g., main, feature/test-utility-script)')
+        choice(name: 'pipelineType', choices: ['build', 'release'], description: 'Pipeline type: build or release')
+    }
+    
+    options {
+        disableConcurrentBuilds()
+        timestamps()
+    }
+
+    stages {
+        stage('Validate Parameters') {
+            steps {
+                script {
+                    echo "==============================="
+                    echo "Deployment Pipeline Parameters"
+                    echo "==============================="
+                    echo "[INFO] Application: ${params.application}"
```

Comment:
I would re-order this by:

Application, Package Type, Build Id, Target Environment.